index.js

```diff
@@ -82,14 +82,6 @@ module.exports = {
         'jest/valid-expect': 'error',
         'jest/valid-title': 'error',
         'max-lines': 'off',
-        'max-lines-per-function': 'off',
```

Comment:
On second thought, we might want to add `max-lines-per-function`, `max-statements`, `max-nested-callbacks`, and `no-undefined ` back in. It is useful in tests to check if something is undefined. It is also useful to nest `describe` and `test` blocks. What do you think?

Reply:
I agree they are useful but they are already set in files in the rules folder which I think covers us - right?

Reply:
This section is the overrides section for test files only. If we want the rule disabled for tests we will need to add it here.

Reply:
Ahh I missed that. And that makes sense to add back in sorry!